src/TestInfrastructure/Helpers/MefTestHelpers.cs

```diff
@@ -24,28 +24,24 @@
 using System.ComponentModel.Composition.Hosting;
 using System.ComponentModel.Composition.Primitives;
 using System.Linq;
+using Moq;
 
 namespace SonarLint.VisualStudio.Integration.UnitTests
 {
     public static class MefTestHelpers
     {
-        public static Export CreateExport<T>(object exportInstance, string contractName = null)
+        public static Export CreateExport<T>() where T: class
```

Comment:
New overload for when we just want a simple Mock of the exported type (which is most of the time).